pjlib/include/pj/assert.h

```diff
@@ -44,11 +44,44 @@
  *
  * @param expr      The expression to be evaluated.
  */
+#ifndef _DEBUG  
+#ifndef pj_assert
+#include "pj/log.h"
+#include <string.h>
+#ifdef _WIN32
```

Comment:
For Windows, it is possible to use MinGW, so need to confirm whether MinGW compiler has `_WIN32` and uses `\\` instead of `/` for path separator.